runtime/lua/vim/treesitter.lua

```diff
@@ -1,4 +1,6 @@
 local a = vim.api
+local query = require'vim.treesitter.query'
+local lang = require'vim.treesitter.language'
```

Comment:
I still feel the same about not liking the `lang` and `language` all within one file -- especially since `language` is the file but you import it as `lang`. I find that confusing